.env

```diff
@@ -26,10 +26,10 @@ MESSENGER_TRANSPORT_DSN=doctrine://default?auto_setup=0
 # Format described at https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
 # IMPORTANT: You MUST configure your server version, either here or in config/packages/doctrine.yaml
 #
-# DATABASE_URL="sqlite:///%kernel.project_dir%/var/data.db"
+DATABASE_URL="sqlite:///%kernel.project_dir%/var/data.db"
 # DATABASE_URL="mysql://db_user:db_password@127.0.0.1:3306/db_name?serverVersion=5.7&charset=utf8mb4"
 # DATABASE_URL="postgresql://symfony:ChangeMe@127.0.0.1:5432/app?serverVersion=13&charset=utf8"
-DATABASE_URL="postgresql://symfony:ChangeMe@database/app?serverVersion=13&charset=utf8"
```

Comment:
Any particular reason for this change? Symfony decided to have PostgreSQL as default in the skeleton/template.

> * [x] This does not alter the source code or configuration

This IS a configuration change. Let's keep issues seperate.